src/components/Projects.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "react"
+import { ProjectCards } from "./lib/ProjectCard"
+import "../assets/stylesheets/index.scss"
+
+export const Projects: FC = () => {
```

Comment:
Remove `{}` and `return`, they are not necessary

Reply:
Remove what?